sdk/dotnet/BranchPolicyBuildValidation.cs

```diff
@@ -23,20 +23,25 @@ namespace Pulumi.AzureDevOps
     /// 
     /// return await Deployment.RunAsync(() =&gt; 
     /// {
-    ///     var exampleProject = new AzureDevOps.Project("exampleProject");
+    ///     var example = new AzureDevOps.Project("example", new()
+    ///     {
+    ///         Name = "Example Project",
```

Comment:
I see many examples where the `Name` property is now used. That's a behavior change since it turns off auto-naming and resources could conflict, isn't it?

Reply:
Yes indeed, this is tracked in https://github.com/pulumi/pulumi-converter-terraform/issues/122 ; the previous behavior was better but we decided not to block on it.